capstone-app/src/app/modules/history/wiki-results.service.ts

```diff
@@ -105,10 +105,10 @@ export class WikiResultsService {
     const paragraphs = parsedText.split('<p>');
     if (paragraphs.length > 9) {
       if (paragraphs[0].startsWith('</h2')) {
-        parsedText = paragraphs.slice(1,10).join('');
+        parsedText = paragraphs.slice(1,10).join('\n');
```

Comment:
I don't think this will work, it will probably just show up as a literal char in the output... plus I made changes today in my branch that addressed this. I would prefer if you change that back to the original ''.

Reply:
Yup I'll remove that as you made changes.

Reply:
Also - just a quick note, if you change some css it renders \n as a newline

Reply:
Oh interesting, I guess since we aren't setting it as html,  I thought that wouldn't work.